### DISCUSSION TITLE
feature request: add a phone number normalizer function

### DISCUSSION THREAD

**Discussion by mohammadizanloo55** [persian-tools/persian-tools#183] [Ideas]
hi, thanks for creating this awesome repository I need a feature Which I will explain below:
`phoneNumberValidator` function accepts different formats 

example :

```
phoneNumberValidator("+989022002580"); // true
phoneNumberValidator("09022002580"); // true
```
But I want to send the phone numbers to Backend only in +98 format and I also want the user who uses my program to be comfortable and enter the phone number in any format he wants and I want to convert it to a format Which Backend accepts
And the thing that came to my mind is that a lot of programmers probably need this feature in their applications.
Here are some examples of how this function works:

```
// This function give the phone number in the first argument and give format Mode in the second argument

phoneNumberNormize("+989022002580", "0"); // 09022002580
phoneNumberNormize("989022002580", "0"); // 09022002580
phoneNumberNormize("09022002580", "0"); // 09022002580
phoneNumberNormize("09022002580", "+98"); //+989022002580
phoneNumberNormize("09802002580", "0"); // Error : this is not valid phone number

```

**Comment by ali-master** [comment]
Hi there, I'll work on it.

**Reply by mediv0** [reply]
> Hi there, I'll work on it.

Do you want me to impl/help with this idea ?

**Reply by ali-master** [reply]
I appreciate your help and if you have time, please implement this feature.

**Comment by mohammadizanloo55** [comment]
@mediv0 
Thank you for creating this function :heart:

**Reply by ali-master** [reply]
We are here to help you and all other developers. This is our mission
